### PR TITLE
Abb 29 MY 24 25

### PR DESCRIPTION
**Pull request by DenisNosnitsin1970**
Abb 29 is a new record (quite complex manuscript), 
MY 24 and 25 only minor changes.

### CODE REVIEW

File: ParisBNF/abb/BNFabb29.xml [hunk 1]
```diff
@@ -0,0 +1,590 @@
...
+            <date>2016-06-07T17:45:12.922+02:00</date>
+         </publicationStmt>
+         <sourceDesc>
+               <msDesc xml:id="ms">
+              <msIdentifier>
+                  <repository ref="INS0303BNF"/>
+                  <collection>Manuscrits orientaux</collection>
+                  <collection>Fonds éthiopien</collection>
+                  <collection>d'Abbadie</collection>
+                <idno facs="https://gallica.bnf.fr/ark:/12148/btv1b10092801m.r=Abbadie%2029?rk=214593;2">BnF Éthiopien d'Abbadie 29</idno>
```
Comment (eu-genia) [inline ParisBNF/abb/BNFabb29.xml:31]: the link should be https://gallica.bnf.fr/ark:/12148/btv1b10092801m (before .)

File: ParisBNF/abb/BNFabb29.xml [hunk 2]
```diff
@@ -0,0 +1,590 @@
+<?xml version="1.0" encoding="UTF-8"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng"
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng"
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="BNFabb29" xml:lang="en" type="mss">
+   <teiHeader xml:base="https://betamasaheft.eu/">
+      <fileDesc>
+         <titleStmt>
+            <title>...</title>
```
Comment (eu-genia) [inline ParisBNF/abb/BNFabb29.xml:7]: title?

File: ParisBNF/abb/BNFabb29.xml [hunk 3]
```diff
@@ -0,0 +1,590 @@
...
+
+                       <physDesc>
+                          <objectDesc form="Codex">
+                             <supportDesc>
+                                <support>
+                                   <material key="parchment"/>
+                                </support>
+                                <extent>
+                                   <measure unit="page">108</measure>
+                                   <measure unit="page" type="blank">12</measure>
```
Comment (eu-genia) [inline ParisBNF/abb/BNFabb29.xml:45]: leaf?

File: ParisBNF/abb/BNFabb29.xml [hunk 4]
```diff
@@ -0,0 +1,590 @@
...
+      </desc>
+      </item>
+      </list>
+      </additions>
+         </physDesc>
+         <history>
+            <origin>
+               <origDate notBefore="1700" notAfter="1799">
+               18th century.</origDate>
+               <ref target="#p2">Unit III</ref> was produced for
```
Comment (eu-genia) [inline ParisBNF/abb/BNFabb29.xml:511]: unit III or p2?

**Review by eu-genia — CHANGES_REQUESTED** [review]

File: ParisBNF/abb/BNFabb29.xml (hunk 1 above)
Comment (DenisNosnitsin1970) [reply ParisBNF/abb/BNFabb29.xml:31]: Thanks! still I do not fullz understand how I can get to this link.

File: ParisBNF/abb/BNFabb29.xml (hunk 2 above)
Comment (DenisNosnitsin1970) [reply ParisBNF/abb/BNFabb29.xml:7]: done

File: ParisBNF/abb/BNFabb29.xml [hunk 5]
```diff
@@ -0,0 +1,590 @@
...
+                       </msContents>
+
+                       <physDesc>
+                          <objectDesc form="Codex">
+                             <supportDesc>
+                                <support>
+                                   <material key="parchment"/>
+                                </support>
+                                <extent>
+                                   <measure unit="page">108</measure>
```
[the remark this answers is not in the document]
Comment (DenisNosnitsin1970) [reply ParisBNF/abb/BNFabb29.xml:44]: leaf.

File: ParisBNF/abb/BNFabb29.xml (hunk 3 above)
Comment (DenisNosnitsin1970) [reply ParisBNF/abb/BNFabb29.xml:45]: leaf.

File: ParisBNF/abb/BNFabb29.xml (hunk 4 above)
Comment (DenisNosnitsin1970) [reply ParisBNF/abb/BNFabb29.xml:510]: done.

**Review by eu-genia — APPROVED** [review]

**Review by thea-m — APPROVED** [review]

**Pull request merged**